holonote/annotate/display.py

```diff
@@ -236,6 +234,15 @@ class AnnotationDisplay(param.Parameterized):
 
     data = param.DataFrame(doc="Combined dataframe of annotation data", constant=True)
 
+    nearest_2d_point_threshold = param.Number(
```

Comment:
```suggestion
    _nearest_2d_point_threshold = param.Number(
```

I am still not a fan of this. We should have better ways to determine if we are clicking on an existing point or a new one. I don't think this will happen in this PR, so let us make it private so we can remove this parameter when we figure out a way to improve it.